### PR TITLE
update usage on README

### PR DESCRIPTION
**Pull request by mandaputtra**
add example on using `.env`.

### CODE REVIEW

**Comment by Eomm** [comment]
Checking the code it should be possible to pass also an object that will be given to the dotenv module, not only a boolean since we use an external lib to do so (i didn't test right now, but should check if dotenv accept a boolean in the config method and not only an object) 

Could you verify also this information and add more details in order to give all the specs to users? 
This fore sure will be amazing useful 👍🏼

**Comment by mandaputtra** [comment]
Okay. yeah will check that out.

**Comment by mandaputtra** [comment]
Turns out there are already test on using `dotenv` options

**Comment by jsumners** [comment]
There is a lot more that could be done to this readme:

1. Even after the given change I had no idea what `dotenv: true` was going to do
1. It is not clear that this module is a wrapper around the `env-schema` module and that any options it supports this module also supports

**Comment by mandaputtra** [comment]
Maybe adding some words like that could be more clear?

**Comment by mandaputtra** [comment]
@Eomm @jsumners  I updated again on last commit what do you think?

Also I found the usage of `dotenv: true` on test, folder not on README.md this is why I created this pull request, I though it will be more efficient when someone know the usage, without looking at the test code.

File: README.md [hunk 1]
```diff
@@ -44,6 +44,23 @@ fastify
   })
 ```
 
+This module is wrapper around [env-schema](https://www.npmjs.com/package/env-schema), to read `.env` file you must pass `dotenv` in options :
```
Comment (Eomm) [inline README.md]:
```suggestion
This module is a wrapper around [env-schema](https://www.npmjs.com/package/env-schema) so, to read `.env` file you must set the `dotenv` in options:
```

**Pull request merged**